config/unavailable_services.yml

```diff
@@ -1,9 +1,8 @@
 services:
   1826:
-    countries:
-      unavailable_in:
-        - Scotland
-        - Northern Ireland
-      content: |
-        <p class="govuk-body">We've matched the postcode to <span class="local-authority">%{local_authority_name}</span>.</p>
-        <p class="govuk-body">This service is not available in %{country_name}. You can find other services on the <span class="local-authority">%{local_authority_name}</span> website.</p>
+    Scotland: |
+      <p class="govuk-body">We've matched the postcode to <span class="local-authority">%{local_authority_name}</span>.</p>
+      <p class="govuk-body">This service is not available in %{country_name}. You can find other services on the <span class="local-authority">%{local_authority_name}</span> website.</p>
```

Comment:
Should we just hardcode the country name in here instead of a variable?

Reply:
Makes perfect sense now the content is country specific - have fixed.